README.txt

```diff
@@ -26,6 +26,9 @@ Install stable/mitaka devstack with GBP.
        ConfiguratorQcow2Image=<configurator image path>
        VyosQcow2Image=<vyos image path>
        HaproxyQcow2Image=<haproxy image path>
+       
+       # if you want to Visibility then give image urls. 
+       VisibilityQcow2Image=
```

Comment:
This is not visibility image. This has both configurator and visibility dockers. For now, we can keep this. Lets discuss and change in next version.